src/sentry/static/sentry/app/redux/reducers/index.js

```diff
@@ -19,6 +19,7 @@ const sharedInitialState = {
 
 export const WORK_BATCH_DEFINITION = 'WORK_BATCH_DEFINITION';
 export const WORK_BATCH = 'WORK_BATCH';
+export const WORK_BATCH_DETAILS = 'WORK_BATCH_DETAILS';
```

Comment:
These constants and createReducer calls should be in the files specific to the resource (e.g. reducers/workBatch.js). I see that constant definitions started sneaking into the index file before this PR, so maybe we should open a JIRA to clean this up later? The index file should really only have the combineReducers call

Reply:
Ok

Reply:
Did you open a ticket or should I?

Reply:
Whoops, I missed the latest change.